sejda-core/src/main/resources/sejda.default.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
-<sejda validation="false">
+<sejda validation="true">
```

Comment:
I think this breaks the default code samples, since the hibernate validation libs are marked as optional, so not included by default.

If validation is on by default, hibernate libs should not be optional (batteries included setup)